README.md

```diff
@@ -23,10 +23,10 @@ To build the latest development sources:
 
 (To build from git, you will need SWIG.)
 
-Once it is installed, `#include <nlopt.h>` in your C/C++ programs and
+Once it is installed, `#include "nlopt.h"` in your C/C++ programs and
```

Comment:
I suppose there is little difference, but it seems weird to use `<...>` for non-system library headers.

http://stackoverflow.com/questions/21593/what-is-the-difference-between-include-filename-and-include-filename